plugins/newsdownloader.koplugin/main.lua

```diff
@@ -388,6 +385,54 @@ function NewsDownloader:setCustomDownloadDirectory()
     }:chooseDir()
 end
 
+function NewsDownloader:changeFeedConfig() 
+    local feed_config_file = io.open(feed_config_path, "rb")
+    local config = feed_config_file:read("*all")
+    feed_config_file:close()
+    config_editor = InputDialog:new{ --todo change to local var, requires fix in save and cancel. No idea how.
```

Comment:
You need to define the local variable first, so it exists and you can reference it then:
```lua
local config_editor
config_editor = InputDialog:new{ [...] -- you can then reference config_editor here in the callbacks }
```

Reply:
never would thought spiting in-line declaration and assignment can make any difference :O
Thanks:)